tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/util/PatternMatcher.java

```diff
@@ -0,0 +1,56 @@
+package com.google.cloud.bigquery.utils.queryfixer.util;
+
+import lombok.AllArgsConstructor;
+import lombok.NonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A helper class that provides static methods to extract substrings from a string based on regular
+ * expression.
```

Comment:
this new line seems really weird :)

Reply:
Rerun the auto format. Seems it looks good.